plugins/meta/flannel/flannel.go

```diff
@@ -92,16 +108,22 @@ func loadFlannelSubnetEnv(fn string) (*subnetEnv, error) {
 			if err != nil {
 				return nil, err
 			}
-			se.mtu = uint(mtu)
+			se.mtu = new(uint)
```

Comment:
you'll get a type mismatch. `mtu` is `uint64` and `se.mtu` is `*uint`. There are other ways to write this code, of course.